frontend/src/container/LogDetailedView/TableView/TableViewActions.tsx

```diff
@@ -100,33 +103,44 @@ export function TableViewActions(
 		);
 	}
 
-	return (
-		<div className={cx('value-field', isOpen ? 'open-popover' : '')}>
-			{record.field === 'body' ? (
-				<CopyClipboardHOC entityKey={fieldFilterKey} textToCopy={textToCopy}>
-					<span
-						style={{
-							color: Color.BG_SIENNA_400,
-							whiteSpace: 'pre-wrap',
-							tabSize: 4,
-						}}
-						dangerouslySetInnerHTML={bodyHtml}
-					/>
-				</CopyClipboardHOC>
-			) : (
-				<CopyClipboardHOC entityKey={fieldFilterKey} textToCopy={textToCopy}>
-					<span
-						style={{
-							color: Color.BG_SIENNA_400,
-							whiteSpace: 'pre-wrap',
-							tabSize: 4,
-						}}
-					>
-						{removeEscapeCharacters(fieldData.value)}
+	let cleanTimestamp: string;
+	if (record.field === 'timestamp') {
+		cleanTimestamp = fieldData.value.replace(/^["']|["']$/g, '');
+	}
+
+	const renderFieldContent = (): JSX.Element => {
+		const commonStyles: React.CSSProperties = {
+			color: Color.BG_SIENNA_400,
+			whiteSpace: 'pre-wrap',
+			tabSize: 4,
+		};
+
+		switch (record.field) {
+			case 'body':
+				return <span style={commonStyles} dangerouslySetInnerHTML={bodyHtml} />;
+
+			case 'timestamp':
+				return (
+					<span style={commonStyles}>
+						{formatTimezoneAdjustedTimestamp(
+							cleanTimestamp,
+							'MM/DD/YYYY, HH:mm:ss.SSS (UTC Z)',
```

Comment:
Create a constant for time formats so that it's easy to use across and is standard.

Reply:
Will make this change separately, Yunus.